includes/discovery/sensors/temperatures/saf.inc.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+if ($device['os'] == 'saf') {
+    $oid  = 'SAF-IPRADIO::sysTemperature';
+    $oids = snmp_get($device, $oid, '-OsqnU', 'SAF-IPRADIO');
+    d_echo($oids."\n");
+
+    if (!empty($oids)) {
+        echo 'SAF Temperature ';
```

Comment:
Agreed! Just based my work to much on previous files from other brands and i should have been trying to improve the code as a whole! Will arrange it ASAP. Same for other comments regarding voltage and temperature discovery files